synapse/models/biz.py

```diff
@@ -96,11 +96,11 @@ def getModelDefs(self):
                     ('title', ('str', {}), {
                         'doc': 'A title for the deal.',
                     }),
-                    ('type', ('biz:dealtype', {}), {
+                    ('type', ('biz:deal:type:taxonomy', {}), {
                         'doc': 'The type of deal.',
                         'disp': {'hint': 'taxonomy'},
```

Comment:
We use 'disp': {'hint': 'taxonomy'}, in a few places but not consistently. It's probably unnecessary if we're updating all the types to be foo:bar:type:taxonomy that include "taxonomy" in the name.

Chance to be consistent & either universally include or universally remove (across all <model>.py files).